mohamedtaham7md/creative-agency-template
# Feature/add gallery

## Implement Gallery section

### Summary of changes
- Added HTML structure for the gallery section and popup functionality.
- Implemented CSS styling to ensure the gallery and popup are responsive and appealing.
- Developed JavaScript functionality to handle the display of images in popup on click,
 including event listener for closing the popup.

### Technical details
- The popup is initially hidden (`display: none`), and show using the class `.show`.
- The implementation is mobile-first with media queries to adjust layout for larger screens.

### Testing Instructions
1. Click on any image in the gallery to open the popup.
2. Verify that the correct image source and alt text are displayed.
3. Test closing the popup using the close button, the Escape key, and by clicking outside the popup.
